app/models/user.rb

```diff
@@ -3,4 +3,7 @@ class User < ActiveRecord::Base
   # :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
+  has_many :books
```

Comment:
This is confusing -- we may need to finesse how a user is linked to books. What if the same user donates and requests?